rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/toolWindow/log/UnityLogPanelModel.kt

```diff
@@ -191,5 +192,17 @@ class UnityLogPanelModel(lifetime: Lifetime, val project: Project, toolWindow: T
         mergeSimilarItems.advise(lifetime) { queueUpdate() }
         project.solution.frontendBackendModel.consoleLogging.lastInitTime.advise(lifetime){ queueUpdate() }
         project.solution.frontendBackendModel.consoleLogging.lastPlayTime.advise(lifetime){ queueUpdate() }
+        project.messageBus.connect(toolWindow.contentManager).subscribe(
+            ToolWindowManagerListener.TOPIC,
+            createToolwindowManagerListener())
+    }
+
+    private fun createToolwindowManagerListener(): ToolWindowManagerListener {
+        return object : ToolWindowManagerListener {
+            override fun toolWindowShown(toolWindow: ToolWindow) {
+                mergingUpdateQueueAction.run()
```

Comment:
Please check whether the `toolWindow` in question is the one you want (like `toolWindow === this@UnityLogPanelModel.toolWindow`, I think?)